apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)